components/brave_wallet_ui/components/buy-send-swap/tabs/swap-tab.tsx

```diff
@@ -185,7 +187,7 @@ function SwapTab (props: Props) {
           accounts={accounts}
           networkList={networkList}
           goBack={goBack}
-          assetOptions={filteredAssetList}
+          assetOptions={isSelectingAsset === 'from' ? swapFromListAssetOptions : filteredAssetList}
```

Comment:
Can you please check if search feature works on from asset list?

Reply:
Search is not working @onyb

Reply:
I think it's because of this change - we need `filteredAssetList` regardless of whether it's a From or a To assets list. See my other comment regarding the change that goes with this one.